src/resources/account/account.helper.js

```diff
@@ -0,0 +1,38 @@
+const psl = require('psl');
```

Comment:
Get rid of account.helper

Add cookie.helper instead, with methods responsible only to adding tokens to cookies (without database access)
#setTokenCookies({ accessToken, refreshToken })
#unsetTokenCookies()


To be able to reuse setTokens/unsetTokens method put them in auth.service